pipelines/optimus/Optimus.wdl

```diff
@@ -87,9 +99,18 @@ workflow Optimus {
           r2_unmapped_bam = FastqToUBam.bam_output,
           whitelist = whitelist
       }
+
+      call FastQC.FastQC as FastQCNoIndex {
+        input:
+          fastq_files = [r1_fastq[index], r2_fastq[index]],
+          limits_file = fastqc_limits,
+          disk = ceil((size(r1_fastq[index], "GiB") + size(r2_fastq[index], "GiB")) * 1.2 + 10)
+      }
     }
 
     File barcoded_bam = select_first([AttachBarcodes.bam_output, AttachBarcodesNoIndex.bam_output])
+    Array[File] fastqc_output_htmls = select_first([FastQC.fastqc_htmls,FastQCNoIndex.fastqc_htmls])
```

Comment:
If you are expecting multiple zips or htmls then I think `select_all()` would be better here.

Reply:
Since only one of `FastQC` and `FastQCNoIndex` will ever be run, using `select_all()` would just result in an extra unnecessary level of array.